ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/caching/Cache.scala

```diff
@@ -38,7 +38,7 @@ object Cache {
 
     override def size: Size = 0
 
-    override private[caching] def entries: Iterable[(Key, Value)] = Iterable.empty
+    override def weight: Size = 0
```

Comment:
Looks like a `val`.

Reply:
Both members here look like they can be `val`s and `NoCache` can be `final`.

Reply:
Good call. Thanks!